wpeverest/user-registration
# Enhancement - Create a menu to show the login form shortcode

### All Submissions:


### Changes proposed in this Pull Request:

This PR is adding the New Login forms tab. It contains all the login-related shortcode. (Design part is pending)

### How to test the changes in this Pull Request:

1. Goto User Registration menu or all form menu
2. Click on the Login forms menu.
3. Verify all the login-related shortcode as well as check the designs.

### Types of changes:


### Other information:


### Changelog entry

> Enhancement - Create a menu to show the login form shortcode.
